notebooks/01_demo.ipynb

```diff
@@ -88,105 +110,416 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Uploading Data"
+    "## Querying Data"
    ]
   },
   {
    "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Make up some data:"
+    "There are three convenience functions in the connector to query data:\n",
```

Comment:
Is it 3 or 2 convenience functions? The list only has 2

Reply:
Ah oops, it was 3 but then I changed it to 2 and didn't update this. Ty!